packages/tasit-contracts/package.json

```diff
@@ -36,7 +36,7 @@
     "@babel/core": "^7.4.0",
     "@babel/node": "^7.2.2",
     "@babel/preset-env": "^7.4.1",
-    "truffle": "^5.0.8",
+    "truffle": "^5.0.14",
     "truffle-hdwallet-provider": "^1.0.0-web3one.5"
```

Comment:
FYI the latest version is `1.0.7`

https://npms.io/search?q=truffle-hdwallet-provider

Reply:
Oh, sure, sorry. I've upgraded and downgraded this packages sometimes trying to solve the CI problem that I've kept it without changes.

Reply:
Ah ok. So do you want to bump it to a higher number?

Reply:
Done.

Reply:
Thanks. Resolving this.